library/core/src/ffi/c_str.rs

```diff
@@ -679,3 +679,35 @@ impl AsRef<CStr> for CStr {
         self
     }
 }
+
+/// Calculate the length of a nul-terminated string. Defers to C's `strlen` when
+/// possible,
```

Comment:
```suggestion
/// possible.
```

(per the [style guide](https://doc.rust-lang.org/nightly/style-guide/index.html#indentation-and-line-width) this could also be one line)